fe1-web/src/features/rollCall/network/RollCallHandler.ts

```diff
@@ -167,14 +167,42 @@ export function handleRollCallCloseMessage(msg: ProcessableMessage): boolean {
 }
 
 /**
- * TODO: Handles a reopen roll call message.
+ * Handles a RollCallReopen message by reopening the corresponding roll call.
  *
- * @param msg
+ * @param msg - The extended message for reopening a roll call
  */
 export function handleRollCallReopenMessage(msg: ProcessableMessage) {
-  console.warn(
-    'A RollCall reopen message was received but its processing logic is not yet implemented:',
-    msg,
-  );
-  return false;
+  if (
+    msg.messageData.object !== ObjectType.ROLL_CALL ||
+    msg.messageData.action !== ActionType.REOPEN
+  ) {
+    console.warn('handleRollCallReopenMessage was called to process an unsupported message', msg);
+    return false;
+  }
+
+  const makeErr = (err: string) => `roll_call/reopen was not processed: ${err}`;
+
+  const storeState = getStore().getState();
+  const lao = getCurrentLao(storeState);
+  if (!lao) {
+    console.warn(makeErr('no LAO is currently active'));
+    return false;
+  }
```

Comment:
conveniently, due to the latest refactorings, you don't need this anymore.
You can now retrieve the `lao.id` from `msg.laoId`.
The code wasn't updated everywhere with this change for lack of time.

Feel free to change that elsewhere in this file too 😄